tests/helpers/wait-for-docker.js

```diff
@@ -0,0 +1,19 @@
+const waitOn = require('wait-on');
+
+console.log("waiting for docker setup to be ready");
+
+var resources = {
+  resources: [
+    'http://localhost:3333/',
+  ],
+  timeout: 30000,
+  followAllRedirects: true,
+  followRedirect: true,
+};
+
+waitOn(resources, error => {
```

Comment:
Since we only use `waitOn` once, I wonder if we can write our own method to check every few seconds instead? It just seems like there's the opportunity here to avoid a dependency.

Reply:
Well, the only way to do this with `node.js` (without adding dependencies) is with the shipped http library. (as `fetch` does not exist). And this module is really low level, so we have to make all the error handling ourselves (even getting the chunks of partial result data is something you have to do yourselves).

So know I rewrote it with the `requests` which is already a dependency of one of the other packages, which means we can drop `wait-on`.